web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
+- Navigating to the metrics page presents users with an empty query field and a prompt to enter a query in the box
+
+![Metrics page with too many results to graph](img/metrics-full.png)
+- If a query cannot be graphed, the results should still be shown in the table below, but the graph field will be populated by an error message until the issue is resolved or the offending query is switched off.
+
+![Metrics page with un-graphable results](img/metrics-error.png)
+- In some cases, we could make specific suggestions about how to improve a given query such as by adding a function
+
+![Query field with autocomplete](img/metrics-autocomplete.png)
+- As the user types, matching metrics and functions appear in a list below. Selecting or clicking on one will fill it into the query field
+
+![Query field with specific autocomplete result](img/metrics-autocomplete-2.png)
+- Results are narrowed as the entered text gets more specific
+
+![Metrics page with a graph of results](img/metrics-results.png)
+- When a valid query is entered, results are graphed and displayed in a table below the query.
```

Comment:
> When a valid query is entered

If the user types something manually, do they need to deselect the field for that validation to occur?

Reply:
The exact behavior here is dependent on a few different implementation details, so I've left it intentionally vague here